src/context.rs

```diff
@@ -31,8 +31,7 @@ pub struct CodegenCx<'gcc, 'tcx> {
     pub codegen_unit: &'tcx CodegenUnit<'tcx>,
     pub context: &'gcc Context<'gcc>,
 
-    // TODO(antoyo): First set it to a dummy block to avoid using Option?
-    pub current_block: RefCell<Option<Block<'gcc>>>,
+    // TODO(bjorn3): First set it to a dummy function to avoid using Option?
```

Comment:
Do you think that's feasible?

Reply:
I have no clue. You know more about gccjit than me.

Reply:
Oh, I meant on the rustc side. It looks like we can probably be in some context where we're not in a function, right?

I guess we can still create a dummy function, but I like the solution of this PR which don't require a dummy block.

Reply:
It may be possible to entirely remove `current_func` from the context in favor of only having it on the builder. I think this would be the best solution if it isn't too much work.